bin/imageServer.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 
 # LSST Data Management System
-# Copyright 2015 AURA/LSST.
+# Copyright 2019 LSST Corporation.
```

Comment:
@kennywlo this change is even worse than before. We do not want to assign copyright to LSST Corp. They are not paying and they didn't do the work. Please follow the guidelines in https://developer.lsst.io/legal/copyright-overview.html?highlight=stanford#copyright-holders

Reply:
@ktlim has clarified this for me.  I will have to update imgserv to the latest DM guidelines, with separate LICENSE and COPYRIGHT files, etc. Thanks.